src/UI.ts

```diff
@@ -46,6 +46,13 @@ const contentHtml = `<!-- HTML Added by UI.ts -->
 		${ButtonImage(coinIcon, "Coin")}
 		${ButtonImage(enemyIcon, "Enemy")}
 	</div>
+	<div class="ui-left ui-editor">
+		<div class="ui-separator"></div>
+		<input id="file-input" type="file" name="name" style="display: none;" />
+		<button class="ui-button" id="save-btn">📥</button>
```

Comment:
Use `${ButtonIcon(<icon:string>, <label:string>)}`. The id is the lowercase label with whitespaces replaced by `-`